data/cartridge.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Cartridge",
+  "description": "Starknet Gaming Console. Play your favorite games.",
+  "short_description": "Starknet Gaming Console",
+  "tags": ["Games"],
+  "contracts": [],
+  "goerliContracts": [],
+  "audits": [],
+  "dotw": false,
+  "links": {
+    "website": "https://cartridge.gg/",
+    "careers": "",
+    "twitter": "https://twitter.com/cartridge_gg",
```

Comment:
```suggestion
    "twitter": "https://twitter.com/cartridge-gg",
```
Fixing github URL

Reply:
@ismaeldm95 the correct Twitter handle is `cartridge_gg`